src/components/index.scss

```diff
@@ -1,14 +1,15 @@
-@import "icon";
-@import "grid";
+@import "Icon";
+@import "Grid";
```

Comment:
Nice! Been wanting to do this for a while. Do these need to be updated in the directory name too?

Reply:
yep, they are, surprised that doesn't show in PR

Reply:
On my branch checked out it shows the files as lowercase, still. But StepNav.scss is updated...